Regression/Core/src/main/java/org/tribuo/regression/ImmutableRegressionInfo.java

```diff
@@ -334,6 +334,23 @@ public String toReadableString() {
         return new ImmutableInfoIterator(idLabelMap);
     }
 
+    @Override
+    public boolean domainAndIDEquals(ImmutableOutputInfo<Regressor> other) {
+        if (size() == other.size()) {
+            for (Map.Entry<Integer,String> e : idLabelMap.entrySet()) {
+                Regressor otherReg = other.getOutput(e.getKey());
+                if (otherReg == null) {
+                    return false;
+                } else if (!otherReg.getDimensionNamesString().equals(e.getValue())) {
```

Comment:
Same as above - does `Regression` keep dimension ordering consistent?

Reply:
Yes, `Regressor` requires that the dimensions are sorted lexicographically. Also it's the same trick as before, the getOutput method returns a single dimension at a time, and they compare by strings.